crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -61,9 +61,8 @@ pub struct AuthorityPerpetualTables {
     /// structure is used to ensure we do not double process a certificate, and that we can return
     /// the same response for any call after the first (ie. make certificate processing idempotent).
     #[default_options_override_fn = "effects_table_default_config"]
-    pub(crate) executed_effects: DBMap<TransactionDigest, TrustedSignedTransactionEffects>,
+    pub(crate) executed_effects: DBMap<TransactionDigest, TransactionEffects>,
```

Comment:
We probably will want to change the name of this table since things in here are not always "executed". If a transaction has been downloaded from state-sync then it will put the effects here, before it has actually been executed. In this case the table name is inaccurate (as is the doc comment for this table)